css/max/demos/view/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Работа функции max() — max() — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
+  <style>
+    html {
+      height: 100%;
+    }
+
+    html {
+      display: grid;
+      place-items: center;
+    }
+
+    body {
+      background-color: #18191c;
+      color: white;
+      font-family: "Roboto Mono", monospace;
+    }
+
+    input {
+      margin: auto;
+      display: block;
+      width: 300px;
+    }
+
+    .rectangle {
+      height: 200px;
+      width: 460px;
+      color: #ffffff;
+      background-color: #2E9AFF;
+      display: grid;
+      font-size: max(4.5vw, 18px);
+      place-content: center;
+      border-radius: 16px;
+      margin: 16px 0;
+    }
```

Comment:
```suggestion
    .rectangle {
      height: 200px;
      width: 460px;
      color: #ffffff;
      background-color: #2E9AFF;
      display: grid;
      font-size: max(4.5vw, 18px);
      place-content: center;
      margin: 16px 0;
    }
```

Скругление не нужно.